src/sessions/SessionManager.ts

```diff
@@ -101,8 +101,20 @@ export class SessionManager {
         this.getSessionFromCookie();
     }
 
+    /**
+     * Returns the sampling decision of a session.
+     *
+     * @returns boolean
+     */
+    public isSampled(): boolean {
+        return this.session.record;
+    }
+
     /**
      * Returns the session ID. If no session ID exists, one will be created.
+     * If a new session is created, it will record the session start event.
+     * If it is created before the page view event is recorded, the page does not exist and the session start event metadata
+     * will not have page attributes.
```

Comment:
nit(blocking): This JavaDoc doesn't adhere to convention. Begin the comment with a [summary fragment](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment). If additional paragraphs are needed, [place a blank line](https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs) between the summary and the next paragraph.

Note that the purpose of the the JavaDoc is to explain the function to the caller. Any implementation details intended to provide context for a maintainer should be placed in the method itself.